tcms/testcases/migrations/0001_initial.py

```diff
@@ -281,10 +281,10 @@ class Migration(migrations.Migration):
                                                             'integrates with the IT system',
                                                   verbose_name='Type')),
                 ('base_url', models.CharField(
-                        max_length=1024, null=True, blank=True,
-                        verbose_name='Base URL',
-                        help_text="""Base URL, for example <strong>https://bugzilla.example.com</strong>!
-Leave empty to disable!
+                    max_length=1024, null=True, blank=True,
+                    verbose_name='Base URL',
+                    help_text="""Base URL, for example
+ <strong>https://bugzilla.example.com</strong>! Leave empty to disable!
```

Comment:
This change is causing the test to fail b/c in the model the newlines in the string are in a different place.